dev/TeachingTip/TeachingTip.cpp

```diff
@@ -25,6 +26,12 @@ winrt::AutomationPeer TeachingTip::OnCreateAutomationPeer()
     return winrt::make<TeachingTipAutomationPeer>(*this);
 }
 
+void TeachingTip::ClosePopupOnUnloadEvent(winrt::IInspectable const& , winrt::RoutedEventArgs const&)
+{
+    IsOpen(false);
+    ClosePopup();
```

Comment:
This means that if the user navigates away from the teaching tip (the unloaded event is fired) we will never play the closing animation. Is this actually the design want?

Reply:
@SavoySchuler